logging/unit_tests/test_connection.py

```diff
@@ -319,17 +322,33 @@ def test_list_sinks_no_paging(self):
         client = _Client(conn)
         api = self._makeOne(client)
 
-        sinks, token = api.list_sinks(self.PROJECT)
+        iterator = api.list_sinks(self.PROJECT)
+        page = six.next(iterator.pages)
+        sinks = list(page)
+        token = iterator.next_page_token
 
-        self.assertEqual(sinks, RETURNED['sinks'])
+        # First check the token.
```

Comment:
I see you have these a few places. Is there a reason to note it more than just having the assert there?